clockwork.go

```diff
@@ -12,6 +12,7 @@ type Clock interface {
 	Sleep(d time.Duration)
 	Now() time.Time
 	Since(t time.Time) time.Duration
+	NewTicker(d time.Duration) Ticker
```

Comment:
This is technically not following the exact definition of `NewTicker` from the stdlib. There it is `func NewTicker(d Duration) *Ticker`. But I don't know if this is a concern